src/playlist.js

```diff
@@ -0,0 +1,66 @@
+var playItem = require('./playitem.js');
+var setupAutoadvance = require('./autoadvance.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var autoadvanceTimeout = null;
+
+  var playlist = function playlist(list) {
+    if (typeof list === 'number') {
+      return player.playlist.item(list);
```

Comment:
I don't think so. For example, you could have the playlist plugin loaded but not actually have a playlist loaded in. You would still have `player.playlist.next` available, it just won't do anything (or error depending on what we want.). The same thing with `play`. It should be available on the player as that is the player's functionality. Just because sources aren't loaded doesn't mean you should remove the function from being available.